src/postProcessing.ts

```diff
@@ -18,15 +18,51 @@ import { SchemaVisitor } from "./postProcessing.types";
  * ```
  */
 export function traverseDocument(swaggerDocument: OpenAPIObject, {
+  operationVisitors = [],
   propertyVisitors = [],
 }: {
+  operationVisitors?: OperationVisitor[],
   propertyVisitors?: SchemaVisitor[],
 } = {}): void {
+  if (operationVisitors?.length) {
```

Comment:
I forgot how the typing exactly works in this case, but those things shouldn't be nullable, right? I don't know if you can remove the `?` in line 24/25 (since the default value will get assigned) or if TS needs it - but then you could probably remove the `?.` in line 27 (like you do with the propertyVisitors below)